deployments/installer/install.ps1

```diff
@@ -0,0 +1,305 @@
+# The following comment block acts as usage for powershell scripts
+# you can view it by passing the script as an argument to the cmdlet 'Get-Help'
+# To view the paremeter documentation invoke Get-Help with the option '-Detailed'
+# ex. PS C:\> Get-Help "<path to script>\install.ps1" -Detailed
+
+<#
+.SYNOPSIS
+    Installs the SignalFx Agent from the package repos.
+.DESCRIPTION
+    Installs the SignalFx Agent from the package repos.  If access_token is not
+    provided, it will prompted for on stdin.  If you want to view full documentation
+    execute Get-Help with the parameter "-Full".
+.PARAMETER access_token
+    The token used to send metric data to SignalFx.
+    .EXAMPLE
+    .\install.ps1 -access_token "ACCESSTOKEN"
+.PARAMETER stage
+    (OPTIONAL) The package stage to install from ['test', 'beta', 'final']. Defaults to 'final'.
+    .EXAMPLE
+    .\install.ps1 -access_token "ACCESSTOKEN" -stage "test"
+.PARAMETER ingest_url
+    (OPTIONAL) Base URL of the SignalFx ingest server.  Defaults to 'https://ingest.signalfx.com'.
+    .EXAMPLE
+    .\install.ps1 -access_token "ACCESSTOKEN" -ingest_url "https://ingest.signalfx.com"
+.PARAMETER api_URL
+    (OPTIONAL) Base URL of the SignalFx API server.  Defaults to 'https://api.signalfx.com'.
+    .EXAMPLE
+    .\install.ps1 -access_token "ACCESSTOKEN" -ingest_url "https://api.signalfx.com"
+.PARAMETER insecure
+    (OPTIONAL) If true then certificates will not be checked when downloading resources Defaults to '$false'.
+    .EXAMPLE
+    .\install.ps1 -access_token "ACCESSTOKEN" -insecure $true
+.PARAMETER package_version
+    (OPTIONAL) Specify a specific version of the agent to install.  Defaults to the latest version available.

```

Comment:
I would probably just call this `agent_version` instead since you don't have any concept of package revisions yet.